bedrock/firefox/templates/firefox/accounts.html

```diff
@@ -100,7 +100,7 @@ <h2 class="c-section-title">
       {% endif %}
     </h2>
 
-    <ul class="c-product-list">
+    <ul class="c-product-list {% if switch('firefox-send-active') %}l-columns-two{% else %}l-columns-three{% endif %}">
```

Comment:
Should we also apply this to the /exp/firefox/accounts/ page? (the CSS changes might need updating there too).